packages/dx-chart-core/src/plugins/axis/computeds.ts

```diff
@@ -105,37 +102,30 @@ export const axisCoordinates: AxisCoordinatesFn = ({
     yText: coordinates,
     text: formatTick(tick),
     ...options,
-  }));
-  return {
-    ticks,
-    sides: [Number(isHor), Number(!isHor)],
-  };
+  });
 };
 
-// It is a part of a temporary walkaround. See note in Axis plugin.
 /** @internal */
-export const createTickFilter = ([width, height]: NumberArray) => (
-  width > 0
-    ? (tick: Tick) => tick.x1 >= 0 && tick.x1 <= width
-    : (tick: Tick) => tick.y1 >= 0 && tick.y1 <= height
-);
-
-const horizontalGridOptions = { y: 0, dy: 1 };
-const verticalGridOptions = { x: 0, dx: 1 };
+export const gridCoordinates: TickCoordinatesFn<Grid> = ({ isHor }) => {
```

Comment:
Seems that it should be `getGridCoordinates` or `gridCoordinatesGetter`?

Reply:
I'll rename method